source/gameDebug/gamefunctions_debug.bmx

```diff
@@ -207,25 +217,32 @@ Type TDebugScreen
 
 
 	Method Render()
+		Local oldCol:SColor8; GetColor(oldCol)
+		Local oldColA:Float = GetAlpha()
+
+		SetColor 0,0,0
+		SetAlpha 0.25 * oldColA
+		DrawRect(0,0, 800, 385)
+		SetColor(oldCol)
+		SetAlpha(oldColA)
+
 		If Not TDebugScreenPage.titleFont
 			TDebugScreenPage.titleFont = GetBitmapFont("default", 12, BOLDFONT)
 			TDebugScreenPage.textFontBold = GetBitmapFont("default", 10, BOLDFONT)
 			TDebugScreenPage.textFont = GetBitmapFont("default", 10)
 		EndIf
 
-		Local oldCol:SColor8; GetColor(oldCol)
-		Local oldColA:Float = GetAlpha()
-
-		TFunctions.DrawOutlineRect(0, 0, sideButtonPanelWidth - 2, 355)
+		TFunctions.DrawOutlineRect(0, 12, sideButtonPanelWidth - 2, 370, new SColor8(100,100,100), new SColor8(0,0,0,150))
 		For Local b:TDebugControlsButton = EachIn sideButtons
 			b.Render()
 		Next
 
-		SetColor 0,0,0
-		SetAlpha 0.2 * oldColA
-		DrawRect(sideButtonPanelWidth,0, 800 - sideButtonPanelWidth, 385)
-		SetColor(oldCol)
-		SetAlpha(oldColA)
+		TDebugScreenPage.textFont.draw("Renderer: "+GetGraphicsManager().GetRendererName(), 3, 355)
```

Comment:
Ich fände es besser, diese Information auf einer der Unterseiten unterzubringen.

Reply:
Kann auch gern hoch in die "Dev-Zeile" (wo fps usw stehen...dann anzeigen, sofern man dev-Osd aktiviert ist).

Da passt es evtl eh besser hin.

Reply:
![image](https://github.com/user-attachments/assets/7cc6c849-b2f8-4032-9775-4b4e6a62b9e2)